### PR TITLE
fix: #id 15143 Error returned is object rather than string.

### PR DESCRIPTION
**Pull request by gerard-ciq**
[Kanban link](https://chartiq.kanbanize.com/ctrl_board/21/cards/15143/details/)

**Description of change**
Testing error message done on object rather than string or array

**Description of testing**

### CODE REVIEW

**Comment by NinjaManatee** [comment]
@gerard-ciq Could you update the testing procedures?

File: gulpfile.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const { launch, connect } = require('hadouken-js-adapter');
 		logToTerminal(`kill: running: ${command}...`);
 
 		exec(command, err => {
-			if (err && !err.includes(error)) {
+			if (err && err.message && !err.message.includes(error)) {
```
Comment (NinjaManatee) [inline gulpfile.js]: Could there be a case where err is a string instead of an Error object?

Comment (gerard-ciq) [reply gulpfile.js]: I don't think so. Unless I am reading wrong (which does happen!) it does not look like it from the docs.

https://nodejs.org/docs/latest-v10.x/api/child_process.html#child_process_child_process_exec_command_options_callback

I checked version 8 too
https://nodejs.org/docs/v0.8.28/api/child_process.html#child_process_child_process_exec_command_options_callback

**Comment by gerard-ciq** [comment]
> @gerard-ciq Could you update the testing procedures?

hahaha.. I was hoping nobody would ask this. I couldn't get an error to occur because I could not get that section to execute (somehow natwest did though). So I had to move the execution somewhere else.

**Pull request closed without merging**